File: example/src/main/kotlin/com/example/_02_structural_patterns/_07_bridge/java/_03_java/BridgeInSpring.java
package com.example._02_structural_patterns._07_bridge.java._03_java;

import org.springframework.jdbc.support.JdbcTransactionManager;
import org.springframework.transaction.PlatformTransactionManager;

public class BridgeInSpring {

    public static void main(String[] args) {
//        MailSender mailSender = new JavaMailSenderImpl();

        PlatformTransactionManager platformTransactionManager = new JdbcTransactionManager();
    }
}

File: example/src/main/kotlin/com/example/_02_structural_patterns/_07_bridge/kt/_03_java/BridgeInSpring.kt
package com.example._02_structural_patterns._07_bridge.kt._03_java

import org.springframework.jdbc.support.JdbcTransactionManager
import org.springframework.transaction.PlatformTransactionManager

fun main() {
//        MailSender mailSender = new JavaMailSenderImpl();
    val platformTransactionManager: PlatformTransactionManager = JdbcTransactionManager()
}
